code/src/App.js

```diff
@@ -1,9 +1,97 @@
-import React from 'react'
+import React, {useState} from 'react' 
+import { Summary } from 'Summary';
+
 
 export const App = () => {
+  const [name, setName] = useState('');
+  const [gender, setGender] = useState('');
+  const [wantsNewsletter, setWantsNewsletter] = useState(false);
+  const [home, setHome] = useState('')
+  const [submitted, setSubmitted] = useState(false);
+  
+  
   return (
-    <div>
-      Find me in src/app.js!
+  <div>{submitted && <Summary name={name} gender={gender} home={home} />}
+    <div className={submitted ? 'formStyleSubmitted' : 'formStyleNotSubmitted'}>
+      {!submitted && ( 
+      <form className="content" onSubmit={event => event.preventDefault()}>
+        <div className="pageTitle">
+          <h1>Home Sweet Home</h1>
+            cat adoption
+        </div>
+        <div className="contacts" >
+          <label className="e-mail">
+            <p>e-mail</p>
+            <input className="formField" 
+              type="email" 
+              placeholder="Example@gmail.com" 
+              onChange={(event)=> setName(event.target.value)} value={name} required/>
```

Comment:
It's good if you have everything in one line **or** on different rows. So this:
```
<input 
   className="formField" 
   type="email" 
   placeholder="Example@gmail.com" 
   onChange={(event)=> setName(event.target.value)} 
   value={name} 
   required
/>
```
or this:
```
<input className="formField" type="email" placeholder="Example@gmail.com" onChange={(event)=> setName(event.target.value)} value={name} required />
```
don't mix the two